wigwam/_docker_distrib.py

```diff
@@ -0,0 +1,59 @@
+from __future__ import annotations
+
+import os
+from textwrap import dedent
+
+from .defaults import install_prefix
+
+
+def distrib_dockerfile(
+    base: str,
+    source_tag: str,
+    source_path: os.PathLike[str] | str,
+    distrib_path: os.PathLike[str] | str,
+    libdir: str,
+) -> str:
+    """
+    Returns a Dockerfile for a distributable build.
```

Comment:
Let's explain what we mean by "distributable" here.

```suggestion
    Returns a Dockerfile for a distributable build.
    
    The distributable image includes the installed software and all of its runtime
    dependencies, but excludes build-time dependencies.
```

Reply:
Let's also mention here in the docstring that the working directory will be changed to `distrib_path` in the resulting image.